### DISCUSSION TITLE
Suggestion: Additional Report Resolution option: Warned w/strike

### DISCUSSION THREAD

**Discussion by betsys** [mastodon/mastodon#30785] [General]
As an admin, my only options for responding to a complaint are to resolve it with no action, which looks as though I am ignoring or dismissing the complaint, or to delete the toot, which I am reluctant to do for a borderline situation.  

Not talking about things that are objectively horrible, such as racism or violence, which of course I would delete.  But there's a lot of gray area, especially around politics. 

I would like an option along the lines of 
Warned: user has been reminded to use content warnings or mark material as sensitive as appropriate. 
             strike recorded locally

The idea is to show that we do read complaints and take them seriously, and to let us keep track of users who might be becoming a problem.

**Comment by ronsipherd** [comment]
Not sure what "strike recorded locally" refers to up there.
Something like "strike one" in baseball?
